harvester/Dockerfile

```diff
@@ -31,4 +31,4 @@ COPY . /app/
 
 RUN cd /app/prepare && SHAPE_RESTORE_SHX=YES python create-buffer.py
 
-CMD python /app/src/run_harvester.py
+CMD  python /app/src/run_daily_weather.py && python /app/src/run_harvester.py
```

Comment:
Aaaah. If the first command fails the second wont be executed. 💥

Reply:
How would you do it? Have to `CMD`s ?

Reply:
Nope. You cant have two commands.

```bash
CMD  python /app/src/run_daily_weather.py ; python /app/src/run_harvester.py
```
Would be one version or put them in a shell script and do some more elaborate error handling

```bash
#!/bin/sh
set -e

python /app/src/run_daily_weather.py || {
    echo "run_daily_weather.py failed";
    # Handle the error or exit
   
}

python /app/src/run_harvester.py
```

Reply:
Just a question. Do they depend on each other? Is the data needed for the harvester or will it be added to the tileset? If not does it have to run in the same process/github action? (separation of concern and so on…)

Reply:
> Do they depend on each other?

no

>  Is the data needed for the harvester 

no

> or will it be added to the tileset?

no, for now it just lands in the `daily_weather_data` table. maybe at a later point it could be in the Mapbox layer.

> does it have to run in the same process

no, maybe it would be better in a separate action. But I don't want to build a whole Docker Image for a separate action just for this simple script. also conceptually, it harvests DWD data (the data behind BrightSky API), so it's not too bad to have it in here.

`CMD  python /app/src/run_daily_weather.py ; python /app/src/run_harvester.py`

i will go with this for now

Reply:
or maybe I have to do some error handling, I want to get notified if either of those commands fail

Reply:
Sorry. Havong exit one on the first command will still not execute the second.